test/Microsoft.AspNet.Diagnostics.Tests/project.json

```diff
@@ -4,6 +4,9 @@
     },
     "dependencies": {
         "Microsoft.AspNet.Diagnostics.Elm": "1.0.0-*",
+        "Microsoft.AspNet.FileProviders.Embedded": "1.0.0-*",
+        "Microsoft.AspNet.StaticFiles": "1.0.0-*",
```

Comment:
Sure, I have removed the test specifically targeting StaticFiles as this scenario is already being covered in the other test that I added. Also added new tests. @Tratcher